### ISSUE TITLE
indexcov --sex ''

### ISSUE THREAD

**Issue by colindaven** [brentp/goleft#41]
Is it somehow possible to disable the sex chromosome plotting or reporting ? Having worked well on mammals I'd like to apply it to bacteria (for global missing regions detection).

Is that in scope ?

Thanks
Colin

**Comment by brentp** [comment]
yes, just use `--sex ''` and it will understand. it defaults to `--sex X,Y` so you have to explicitly give it the empty string.

**Comment by brentp** [comment]
any chance you could share the bam and reference privately?

**Comment by brentp** [comment]
actually just the bai and anonymized bam and reference.
see: https://github.com/brentp/goleft/releases/tag/v0.1.10
to get a program to make a minimal bam.

**Comment by brentp** [comment]
also, from the lines of your error, it looks like you might have an older version of goleft. can you try the latest version?

**Comment by colindaven** [comment]
Pretty sure it's the latest release from march 15
```
 Mar 15 16:17 goleft_linux64*


 ./goleft_linux64 
goleft Version: 0.1.18

covstats   : coverage stats across bams by sampling
depth      : parallelize calls to samtools in user-defined windows
depthwed   : matricize output from depth to n-sites * n-samples
indexcov   : quick coverage estimate using only the bam index
indexsplit : create regions of even coverage across bams/crams
samplename : report samplename(s) from a bam's SM tag

```

**Comment by brentp** [comment]
can you share a test-case for this?

**Comment by colindaven** [comment]
Ok, cheers for letting me know
